packages/cysync-core/src/dialogs/AddAccountGuide/Dialogs/Asset/SelectCryptoDialog.tsx

```diff
@@ -0,0 +1,178 @@
+import {
+  LangDisplay,
+  DialogBox,
+  DialogBoxHeader,
+  DialogBoxBody,
+  Typography,
+  Image,
+  Container,
+  DialogBoxFooter,
+  Button,
+  Dropdown,
+  addIcon,
+  bitcoinIcon,
+  etheriumBlueIcon,
+  DropDownListItemProps,
+  walletIcon,
+} from '@cypherock/cysync-ui';
+import { binanceIcon, solanaIcon } from '@cypherock/cysync-ui/src';
```

Comment:
Do not import from `src`

```suggestion
import { binanceIcon, solanaIcon } from '@cypherock/cysync-ui';
```

Reply:
changed